### ISSUE TITLE
Unify /suspend and /idle commands

### ISSUE THREAD

**Issue by raptor** [bitfighter/bitfighter#186]
_From [watusim...@bitfighter.org](https://code.google.com/u/105427273526970468779/) on December 15, 2012 12:55:00_

If you are the only player, and you issue the /idle command, it has an effect very similar to the /suspend command.

These should be unified, and, perhaps, /suspend should be removed from the game.

The game should automatically suspend if:
* there are 0 players
* all players are /idle or otherwise spawn delayed

This already happens I think, just putting it down as a note

_Original issue: http://code.google.com/p/bitfighter/issues/detail?id=186_

**Comment by raptor** [comment]
_From [watusim...@bitfighter.org](https://code.google.com/u/105427273526970468779/) on December 15, 2012 11:55:17_

**Labels:** 019

**Comment by raptor** [comment]
_From [buckyballreaction](https://code.google.com/u/buckyballreaction/) on June 16, 2013 19:35:09_

I've been in the suspend code for the past hour - this behavior you requested is there.  /suspend has been removed and if all players are /idle, the server will suspend automatically, just like if it had 0 players.

I did notice one thing: if you are the host and do /idle the server is suspended.  I think this is because the mTimeToSuspend is used for both /idle and server suspend.  Do we want to separate these?

**Status:** Verified  
  **Owner:** buckyballreaction
